src/topology/continuous_function/algebra.lean

```diff
@@ -149,6 +154,17 @@ rfl
 -- don't make `zsmul_comp` simp as the linter complains it's redundant WRT `smul_comp`
 attribute [simp] zpow_comp
 
+section comp_translate
+-- This arguably doesn't quite belong here because it is about algebraic structures on the source,
+-- not target, of the maps; but it does have the same import dependencies as the other stuff here.
+
+/-- The continuous map `λ x, f (x + m)`. -/
+@[to_additive "The continuous map `λ x, f (x * m)", simps]
+def comp_mul_right [has_mul α] [has_continuous_mul α] (f : C(α, β)) (m : α) : C(α, β) :=
+f.comp (mk _ (continuous_mul_right m))
```

Comment:
See https://github.com/leanprover-community/mathlib/pull/18392#discussion_r1111194246

Reply:
@eric-wieser, re your comment here: https://github.com/leanprover-community/mathlib/pull/18392#discussion_r1111244305. I'm saying that we should have `continuous_map.mul_right` but not `continuous_map.comp_mul_right`, so I think we agree.